public/app/features/snapshot/snapshot_ctrl.ts

```diff
@@ -0,0 +1,43 @@
+///<reference path="../../headers/common.d.ts" />
+
+import angular from 'angular';
+import _ from 'lodash';
+
+export class SnapshotsCtrl {
+
+  /** @ngInject */
+  constructor(backendSrv, $scope) {
+    $scope.init = function() {
```

Comment:
I tried the way you are telling but then I got stuck at this.AppEvent. No worries, will try once more.